index.html

```diff
@@ -2,31 +2,12 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 
 <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```

Comment:
This tag is necessary

Reply:
<meta charset="UTF-8"> is used in html5. The one above is html 4.0.1

Reply:
Have you added this `<meta charset="UTF-8">` in the index file?